src/schema/posts.ts

```diff
@@ -216,6 +216,11 @@ export const typeDefs = /* GraphQL */ `
     """
     read: Boolean
 
+    """
+    Whether the post should be visible or not
+    """
+    visible: Boolean
```

Comment:
I'm not sure we should propagate this field down to the client. Thoughts? cc @rebelchris